### PR TITLE
initialChoice is use in resultPerPages

### PR DESCRIPTION
**Pull request by orobert-coveo**
https://coveord.atlassian.net/browse/JSUI-2390

When the default value of ResultsPerPage was in ChoiceDisplayed, it didn't use the initial choice option

[Deploy](https://dashboard.heroku.com/pipelines/a3535101-5bbf-4a5b-a909-47fcf8c9f149)

### CODE REVIEW

File: unitTests/ui/ResultsPerPageTest.ts [hunk 1]
```diff
@@ -213,6 +213,19 @@ export function ResultsPerPageTest() {
         });
         expect(test.env.queryController.options.resultsPerPage).toBe(firstChoice);
       });
+
+      it('initialChoice set to non default value is use when default value is in choicesDisplayed', () => {
```
Comment (samisayegh) [inline unitTests/ui/ResultsPerPageTest.ts]: I think the description could be clearer. It should mention that the initialChoice value is set to the configured value when the QSM value is the default value. e.g.

```
it (`when the QSM numberOfResults is equal to the default QSM number of results,
when the initialChoice option is not equal to the default QSM numberOfResults,
it sets the resultsPerPage to the initialChoice`, () => { ... })
```

File: unitTests/ui/ResultsPerPageTest.ts [hunk 2]
```diff
@@ -213,6 +213,19 @@ export function ResultsPerPageTest() {
...
+        let initialChoice = 5;
```
Comment (samisayegh) [inline unitTests/ui/ResultsPerPageTest.ts]: It's a good practice to always use `const` when the variable value is not modified.

File: unitTests/ui/ResultsPerPageTest.ts (hunk 1 above)
Comment (orobert-coveo) [reply unitTests/ui/ResultsPerPageTest.ts]: just to be sure but QSM is the QueryStateModel?

Comment (samisayegh) [reply unitTests/ui/ResultsPerPageTest.ts]: Yep,

Comment (orobert-coveo) [reply unitTests/ui/ResultsPerPageTest.ts]: ok thx

**Pull request merged**